vdf/src/lib.rs

```diff
@@ -43,7 +43,7 @@
 //!
 //! ### To use the VDF library
 //!
-//! ```rust
+//! ```rust,no_run
```

Comment:
Would it be possible to extend the example instead, so that it validates the proof instead of comparing to some known binary value?

Reply:
done

Reply:
Thanks! I guess the `no_run` can be removed again?